app/src/main/java/org/mozilla/webmaker/web/javascript/WebmakerAPI.java

```diff
@@ -1,22 +1,21 @@
 package org.mozilla.webmaker.web.javascript;
 
-import android.util.Log;
-
 import org.mozilla.webmaker.BaseActivity;
-import org.mozilla.webmaker.WebmakerActivity;
 import org.mozilla.webmaker.view.WebmakerWebView;
 import org.xwalk.core.JavascriptInterface;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 
-/**
- * ...
- */
 public class WebmakerAPI {
 
-    // Static behaviour
+    // used for triggering JS on the main UI thread
+    protected WebmakerWebView currentView = null;
+    protected BaseActivity currentActivity = null;
 
+    protected Queue queue;
+
+    // Static behaviour
```

Comment:
this was organized as "statics first, then instance", so if we're going to mix the two, the comments about which block starts where is no longer really relevant, I'd recommend keeping the static vs instance functionality grouped rather than mixed, since the two blocks aren't supposed to casually interact.